articles/virtual-desktop/set-up-service-alerts.md

```diff
@@ -40,11 +40,11 @@ To configure service alerts:
 
 1. Sign in to the [Azure portal](https://portal.azure.com/).
 2. Select **Service Health.**
-3. Follow the instructions in [Create activity log alerts on service notifications](../service-health/alerts-activity-log-service-notifications-portal.md?toc=%2fazure%2fservice-health%2ftoc.json) to set up your alerts and notifications.
+3. Follow the instructions in [Create activity log alerts on service notifications](../service-health/alerts-activity-log-service-notifications-portal.md?toc=%2fazure%2fservice-health%2ftoc.json) to set up your alerts and notifications. Note, Azure Virtual Desktop is listed as Windows Virtual Desktop within the Services drop down box when creating an alert rule.
```

Comment:
```suggestion
3. Follow the instructions in [Create activity log alerts on service notifications](../service-health/alerts-activity-log-service-notifications-portal.md?toc=%2fazure%2fservice-health%2ftoc.json) to set up your alerts and notifications. 

   >[!NOTE]
   >Azure Virtual Desktop is listed as "Windows Virtual Desktop" in the drop-down menu under **Services** when creating an alert rule.
```

Reply:
@TomHickling I've made a few tweaks to the pull request. Please let me know when you've made these edits, and I'll check if they look okay in the build and sign off on them.

Reply:
Looks good